src/types/UserResponce.ts

```diff
@@ -0,0 +1,10 @@
+export interface UserResponce {
+  id: number;
+  createdAt: string;
+  email: string;
+  name: string;
+  phone: string;
+  updatedAt: string;
+  username: string;
+  website: null | unknown;
```

Comment:
For the 'website' property, use 'string|null' instead of 'null|unknown'.

Reply:
updated